06-building-a-program-start-to-finish/writing-tests/Output/inverted-index-spec.js

```diff
@@ -0,0 +1,171 @@
+/* global describe, it, expect */
+const valid1 = require('../../src/test books/valid1.json'),
+      valid2 = require('../../src/test books/valid2.json'),
+      valid3 = require('../../src/test books/valid3.json'),
+      valid4 = require('../../src/test books/valid4.json'),
+      invalid1 = require('../../src/test books/invalid1.json'),
+      invalid2 = require('../../src/test books/invalid2.json'),
+      invalid3 = require('../../src/test books/invalid3.json'),
+      invalid4 = require('../../src/test books/invalid4.json'),
+      invalid5 = require('../../src/test books/invalid5.json'),
+      invalid6 = require('../../src/test books/invalid6.json'),
+      invalid7 = require('../../src/test books/invalid7.json');
+
```

Comment:
Please renames these files based on the content. File names should be in camelCase.